test/dlc_tests/ec2/test_efa.py

```diff
@@ -8,16 +8,23 @@
     LOGGER,
     get_account_id_from_image_uri,
     get_region_from_image_uri,
+    get_framework_and_version_from_tag,
     is_pr_context,
     is_efa_dedicated,
     login_to_ecr_registry,
     run_cmd_on_container,
 )
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+
 from test.test_utils.ec2 import get_efa_ec2_instance_type, filter_efa_instance_type
 
 BUILD_ALL_REDUCE_PERF_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "efa", "build_all_reduce_perf.sh")
 EFA_SANITY_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "efa", "testEFASanity")
-EFA_INTEGRATION_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "efa", "testEFA")
+EFA_NCCL_TESTS_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "efa", "test_nccl_tests_using_efa")
+EFA_AWS_OFI_NCCL_PLUGIN_TEST_CMD = os.path.join(
+    CONTAINER_TESTS_PREFIX, "efa", "test_aws_ofi_nccl_plugin_using_efa"
+)
```

Comment:
The TensorFlow EFA test in this script also uses `EFA_INTEGRATION_TEST_CMD`, and needs to be modified to use these two new EFA NCCL Test commands instead.